packages/@divvi/mobile/src/swap/SwapScreenV2.test.tsx

```diff
@@ -1342,21 +1356,147 @@ describe('SwapScreen', () => {
       fromTokenIsImported: false,
       amount: '10',
       amountType: 'sellAmount',
-      allowanceTarget: defaultQuote.unvalidatedSwapTransaction.allowanceTarget,
-      estimatedPriceImpact: defaultQuote.unvalidatedSwapTransaction.estimatedPriceImpact,
-      price: defaultQuote.unvalidatedSwapTransaction.price,
-      provider: defaultQuote.details.swapProvider,
+      allowanceTarget: quoteCeloToCusd.unvalidatedSwapTransaction.allowanceTarget,
+      estimatedPriceImpact: quoteCeloToCusd.unvalidatedSwapTransaction.estimatedPriceImpact,
+      price: quoteCeloToCusd.unvalidatedSwapTransaction.price,
+      provider: quoteCeloToCusd.details.swapProvider,
       web3Library: 'viem',
       gas: 1821000,
       maxGasFee: 0.021852,
-      maxGasFeeUsd: 0.28529642665785426,
+      maxGasFeeUsd: 0.01770012,
       estimatedGasFee: 0.014568,
-      estimatedGasFeeUsd: 0.19019761777190283,
+      estimatedGasFeeUsd: 0.01180008,
       feeCurrency: undefined,
       feeCurrencySymbol: 'CELO',
       txCount: 2,
       swapType: 'same-chain',
       swapId: mockSwapId,
+      isUnfavorableRate: false,
+    })
+  })
+
+  describe.each([
```

Comment:
These are the new unit tests. Other tests are changes related to making exchange rate / prices consistent so they don't trigger the new bottom sheet.